src/ray/raylet/node_manager.h

```diff
@@ -43,8 +43,8 @@ struct NodeManagerConfig {
   int node_manager_port;
   /// The initial number of workers to create.
   int num_initial_workers;
-  /// The number of workers per process.
-  int num_workers_per_process;
+  /// The number of workers per process per language.
+  std::unordered_map<Language, int, std::hash<int>> num_workers_per_process_by_lang;
```

Comment:
you can use the `EnumUnorderedMap` util class to get rid of the `std::hash<int>` part.
But I think it's also fine to just remove this filed from `NodeManagerConfig` and just let `WorkerPool` read the config from `RayConfig::instance()`.

Reply:
Changed to read config in `WorkerPool`.